Softeq.XToolkit.WhiteLabel.Forms/Navigation/IMasterDetailViewModel.cs

```diff
@@ -0,0 +1,13 @@
+// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using Softeq.XToolkit.WhiteLabel.Mvvm;
+
+namespace Softeq.XToolkit.WhiteLabel.Forms.Navigation
+{
+    public interface IMasterDetailViewModel
+    {
+        ViewModelBase MasterViewModel { get; }
```

Comment:
Will be better to use `IViewModelBase` instead of `ViewModelBase` in the navigation context at all.